Runbooks/startstop/1.1/ScheduledStartStop_Base_Classic.ps1

```diff
@@ -0,0 +1,161 @@
+﻿<#
+.SYNOPSIS
+	This runbook used to perform action start or stop in classic VM group by Cloud Services
+.DESCRIPTION
+	This runbook used to perform action start or stop in classic VM group by Cloud Services
+	This runbook requires the Azure Automation Run-As (Service Principle) account, which must be added when creating the Azure Automation account.
+ .EXAMPLE
```

Comment:
Extra space at start of line

Reply:
Fixed something. It is not important anyway.